docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -339,6 +339,7 @@ Used to lookup fungible assets balances
     * `eip155:8508132` - Scroll
     * `eip155:7854577` - Zora
     * `eip155:1313161554` - Aurora
+* `forceUpdate` - (Optional) Token contract address to force the latest balance update on it. `chain` must be also provided for this type of call.
```

Comment:
Would it be possible to pass array of addresses as `forceUpdate` ? In swaps, we are updating two tokens' balance when swapping, like MATIC -> AVAX, we need to force update both of their balance

Reply:
Updated to pass a list of comma separated addresses.